data/formats-data.js

```diff
@@ -5187,7 +5194,7 @@ exports.BattleFormatsData = {
 		tier: "UU",
 	},
 	larvesta: {
-		randomBattleMoves: ["flareblitz", "uturn", "wildcharge", "zenheadbutt", "morningsun", "willowisp"],
+		randomBattleMoves: ["flareblitz", "uturn", "wildcharge", "morningsun", "willowisp"],
```

Comment:
zen headbutt is run as a scarf set

Reply:
id rather util

Reply:
util generally doesnt run wild charge

Reply:
indeed but its coverage is useful nevertheless. the current pool will give you 2-3 damaging attacks per set which is just the range im looking for on this mon